JSDemos/Demos/Toolbar/Multiline/jQuery/index.js

```diff
@@ -0,0 +1,390 @@
+const toolbarDataDefault = [
+  {
+    location: 'before',
+    widget: 'dxButton',
+    options: {
+      icon: 'undo',
+      onClick() {
+        DevExpress.ui.notify('Undo button has been clicked!');
+      },
+    },
+  },
+  {
+    location: 'before',
+    widget: 'dxButton',
+    options: {
+      icon: 'redo',
+      onClick() {
+        DevExpress.ui.notify('Redo button has been clicked!');
+      },
+    },
+  },
+  {
+    cssClass: 'toolbar-separator-container',
```

Comment:
We use this separator several times. Can we mode its config to a separate object to reduce copypaste?